src/org/rust/lang/RustSdkType.kt

```diff
@@ -67,10 +85,22 @@ class RustSdkType : SdkType("Rust SDK") {
         }
 
         try {
-            val cmd = GeneralCommandLine()
+            val cmd = if (isMultiRust(sdkHome)) {
+                val sdkLibraryPath = getSdkLibrary(sdkHome).absolutePath
+                GeneralCommandLine()
+                    .withWorkDirectory(sdkHome)
+                    .withEnvironment(hashMapOf(
+                        "DYLD_LIBRARY_PATH" to sdkLibraryPath,
+                        "LD_LIBRARY_PATH" to sdkLibraryPath
+                    ))
+                    .withExePath(rustc.absolutePath)
+                    .withParameters("-C rpath", "--version")
```

Comment:
I suppose *every* caller of cargo/rustc will probably need those parameters, right? So if the SDK was used by some other parts of the IntelliJ framework it would always need those. I'm not sure how that would be handled best.

/cc @atsky